src/screens/MyPromotions.tsx

```diff
@@ -202,20 +263,19 @@ export default function MyPromotions(): ReactElement {
               // TODO: https://promopal.atlassian.net/browse/PP-96
               restaurantName=""
               schedules={promotion.schedules}
+              votes={promotion.votes}
+              voteState={promotion.voteState}
               onDeleteButtonClick={() => onDeleteButtonClick(promotion)}
+              onDownvoteClick={() => onDownvoteClick(promotion.id)}
               onSaveButtonClick={() => onSaveButtonClick(promotion.id)}
+              onUpvoteClick={() => onUpvoteClick(promotion.id)}
             />
           </Col>
         ))}
       </Row>
     );
   };
 
-  // TODO: https://promopal.atlassian.net/browse/PP-80
```

Comment:
Do we maybe want to do what we do in MyAccount with the `return <Redirect to="/account" />;` ? Although it's no big deal since this should be happening.